src/dstack/_internal/core/models/backends/base.py

```diff
@@ -9,6 +9,7 @@ class BackendType(str, enum.Enum):
     AZURE = "azure"
     DSTACK = "dstack"
     GCP = "gcp"
+    DATACRUNCH = "datacrunch"
```

Comment:
You also have to patch:

* `core/models/backends/__init__.py`
* `server/services/backends/__init__.py`
* `server/services/backends/configurators/datacrunch.py`
* `server/services/backends/config.py`